web/locales/fi.json

```diff
@@ -0,0 +1,1786 @@
+    },
+    "mobile-main-menu": {
+        "map": "electricityMap",
+        "areas": "Alueet",
+        "about": "Tietoa"
+    },
+    "onboarding-modal": {
+        "view1": {
+            "subtitle": "Sähkön ilmastovaikutusten kartoittaminen"
+        },
+        "view2": {
+            "header": "Katso, kuinka paljon hiilidioksidia vapautuu sähkön tuottamiseksi, reaaliajassa.",
+            "text": "Värjäämme alueet ympäri maailmaa niiden kuluttaman sähkön hiili-intensiteetin perusteella (mukaan lukien kaikki kasvihuonekaasut). Mitä vihreämpi väri, sitä ilmastoystävällisempi sähkö."
+        },
+        "view3": {
+            "header": "Tiedä mistä sähkösi tulee",
+            "text": "Kartan nuolet osoittavat sähkövirrat eri alueiden välillä. Napsauta aluetta nähdäksesi lisätietoja sen sähkön alkuperästä."
+        },
+        "view4": {
+            "header": "Reaaliaikaiset tuuli- ja aurinko-olosuhteet",
+            "text": "Tuuli- ja aurinkoenergian tuotanto riippuvat säästä. Paina aurinko- ja tuulinäppäimiä nähdäksesi tuulen ja auringonpaisteen reaaliaikaisen voimakkuuden ympäri maailmaa."
+        }
+    },
+    "country-panel": {
+        "source": "Lähde",
+        "carbonintensity": "Hiili-intensiteetti",
+        "lowcarbon": "Vähähiilinen",
+        "renewable": "Uusiutuva",
+        "electricityproduction": "Sähkön tuotanto",
+        "electricityconsumption": "Sähkön kulutus",
+        "bysource": "alkuperä",
+        "emissions": "Hiilidioksidipäästöt",
+        "addeditsource": "<a href=\"%s\" target=\"_blank\">lisää tai muokkaa lähdettä</a>",
+        "helpfrom": "heidän avustuksella",
+        "noDataAtTimestamp": "Tiedot eivät ole väliaikaisesti käytettävissä valitulle ajankohdalle",
+        "noLiveData": "Live-tiedot eivät ole väliaikaisesti käytettävissä tällä alueella",
+        "noParserInfo": "Meillä ei ole vielä tietoa tästä alueesta.<br/> Haluatko auttaa tämän korjaamisessa? Voit avustaa <a href=\"%s\" target=\"_blank\">lisäämällä alueen tiedot</a>.",
+        "now": "Now"
+    },
+    "left-panel": {
+        "zone-list-header-title": "Ilmastovaikutukset alueittain",
+        "zone-list-header-subtitle": "Sijoitus kulutetun sähkön hiili-intensiteetin mukaan (gCO₂eq/kWh)",
+        "search": "Etsi alueita"
+    },
+    "country-history": {
+        "carbonintensity24h": "Hiili-intensiteetti viimeisen 24 tunnin aikana",
+        "emissions24h": "Päästöt viimeisen 24 tunnin aikana",
+        "emissionsorigin24h": "Päästöjen alkuperä viimiesen 24 tunnin aikana",
+        "emissionsproduction24h": "Päästöt tuotettu viimiesen 24 tunnin aikana",
+        "electricityorigin24h": "Sähkön alkuperä viimeisen 24 tunnin aikana",
+        "electricityproduction24h": "Sähkön tuotanto viimeisen 24 tunnin aikana",
+        "electricityprices24h": "Sähkön hinnat viimeisen 24 tunnin aikana",
+        "Getdata": "Saa historiallista tietoa, marginaalinen ja ennuste API"
```

Comment:
An alternative I have is `Hanki pääsy historiallisiin tietoihin, sekä marginaali- ja ennuste-API:hin` but that won't fit in the UI

![image](https://user-images.githubusercontent.com/13119842/83305369-9d24f000-a1ce-11ea-98b5-e712b6d27dd2.png)

Reply:
@desentso any suggestion?

Reply:
That's quite a tricky one to translate, I think the alternative you proposed sounds good.

It could be slightly shortened by changing it to: `Hanki pääsy historiatietoihin, sekä marginaali- ja ennuste-API:hin`. Would that fit in the UI?

Reply:
Seems like that doesn't fit the UI either.

We could shorten it even more by chaining the historical data to the API as well `Hanki pääsy historia-, marginaali- ja ennuste-API:hin`, that would fit in the UI as well.

![electricitymap](https://user-images.githubusercontent.com/12684920/83974496-4cac4180-a8f6-11ea-94b1-9ebd13e313e0.PNG)

Reply:
@desento I like that, but it implies you get access to a "historical API" instead of "historical data", which is not technically correct. I've made the change for now as I couldn't come up with a better text myself.

Reply:
Yes, but to me, it seems like there are also historical API endpoints, e.g. http://static.electricitymap.org/api/docs/index.html#past-carbon-intensity-history instead of a historical data set, though the English text is misleading if that's actually the case.